### ISSUE TITLE
symbolic_computation_speedups doesn't run

### ISSUE THREAD

**Issue by gftabor** [symforce-org/symforce#204]
**Describe the bug**
https://github.com/symforce-org/symforce/blob/main/notebooks/symbolic_computation_speedups.ipynb
throws an error
```
AttributeError: module 'symforce.symbolic' has no attribute 'Eq'
```
**To Reproduce**
Run it

**Environment (please complete the following information):**
 - OS and version:Ubuntu Linux 20.04
 - Python version 3.9
 - SymForce Version 0.5.0

**Comment by aaron-skydio** [comment]
Thanks for reporting!  Should be fixed by #205 , let me know if that doesn't work

**Comment by gftabor** [comment]
Thanks I tried to find it myself but "eq" is not a good keyword to search for.
